app/views/sign_up/completions/show.html.erb

```diff
@@ -35,6 +35,15 @@
                 last_number: attribute_value[-1],
               ),
             ) %>
+      <% elsif attribute_key == :email && IdentityConfig.store.feature_select_email_to_share_enabled %>
+        <div class="display-flex flex-justify">
+          <%= attribute_value.to_s %>
+            <% if @presenter.multiple_emails? %>
+              <p class='font-body-2xs text-right'><%= link_to t('help_text.requested_attributes.change_email_link'), sign_up_select_email_path %></p>
+            <% else %>
+              <p class='font-body-2xs text-right'><%= link_to t('account.index.email_add'), add_email_path %></p>
+            <% end %>
```

Comment:
Nit:

- The indentation is off by one tab level here.
- If both paths of the condition render the same paragraph wrapping element, maybe we move the condition inside the paragraph?

```suggestion
          <p class="font-body-2xs text-right">
            <% if @presenter.multiple_emails? %>
              <%= link_to t('help_text.requested_attributes.change_email_link'), sign_up_select_email_path %>
            <% else %>
              <%= link_to t('account.index.email_add'), add_email_path %>
            <% end %>
          </p>
```

Reply:
Cleaned up this markup.